src/ui/MessageContent/index.scss

```diff
@@ -4,8 +4,7 @@
   display: inline-flex;
   flex-direction: row;
   position: relative;
-  width: calc(100% - 48px);
-  padding: 0 24px;
+  width: 100%;
```

Comment:
doesn't it need to add this?
```
box-sizing: border-box;
```

Reply:
No it looks fine without it. maybe because of padding in MessageList?